README.md

```diff
@@ -37,29 +39,72 @@ Install `Docker` according to the [official documentation](https://docs.docker.c
 
 ## Usage
 
-Generate a new Project using this:
+Generate a new Plone 6 Project:
 
 ```shell
 cookiecutter gh:collective/cookiecutter-plone-starter
 ```
 
+`Cookiecutter` generates a file structure.
+
+For an initial build of backend and frontend:
+
+```shell
+make install
+```
+
+In two separate terminal sessions, start backend and frontend:
+
+To start backend:
+
+```shell
+make start-backend
+```
+
+To start frontend:
+
+```shell
+make start-frontend
+```
+
+After changes please re-build both with
+
+```shell
+make build
+```
+
+and restart backend and frontend by stopping and re-running
```

Comment:
Shall we add it can be stopped with `Ctrl-c`?

Reply:
yes, definitly.